scu-de/src/fiskaltrust.Middleware.SCU.DE.SwissbitCloudV2/Models/ExportStateResponseDto.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace fiskaltrust.Middleware.SCU.DE.SwissbitCloudV2.Models
+{
+    public class ExportStateResponseDto
+    {
+        /* 
```

Comment:
If you use XML comments, the IDE will show it - not mandatory obviously, but nicer in my opinion :)

```cs
        /// <summary>
        /// Possible states: "pending", "success", "failure"
        /// </summary>
```